managed/devops/opscli/ybops/cloud/aws/cloud.py

```diff
@@ -387,5 +388,16 @@ def start_instance(self, args):
             instance = ec2.Instance(id=args["id"])
             instance.start()
             instance.wait_until_running()
+            # The OS boot up may take some time,
+            # so retry untill the instance allows SSH connection to it via port 22.
+            retry_count = 0
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            while retry_count < 25:
```

Comment:
Usually, it takes 5 to 10 seconds, please suggest pessimistic retry count here @Arnav15

Reply:
I think we can put a 10 second sleep between each iteration, that should be fine. And giving it 5 minutes (so 30 iterations) should also be good enough.